ballsdex/packages/admin/info.py

```diff
@@ -129,6 +129,26 @@ async def user(
             catch_date__gte=datetime.datetime.now() - datetime.timedelta(days=days),
             player=player,
         )
+        class recentlyCaughtButton(discord.ui.View):
```

Comment:
```suggestion
        class recentlyCaughtButton(discord.ui.View):
        class RecentlyCaughtButton(discord.ui.View):
```

classes should be named using camelcase, just the standard used across python

Reply:
recentlyCaughtButton is already correct camelcase, which allows the first letter of the name to be either case. I believe you are describing PascalCase, which requires the first letter of each word to be only capital. I will change it accordingly.